app/connectors/VatObligationsConnector.scala

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package connectors
+
+import java.time.LocalDate
+
+import config.AppConfig
+import connectors.httpParsers.ResponseHttpParsers.HttpGetResult
+import javax.inject.{Inject, Singleton}
+import models.Obligation.Status
+import models.VatReturnObligations
+import play.api.Logger
+import play.api.http.HeaderNames
+import services.MetricsService
+import uk.gov.hmrc.http.{HeaderCarrier, HttpReads}
+import uk.gov.hmrc.play.bootstrap.http.HttpClient
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class VatObligationsConnector @Inject()(http: HttpClient,
+                                        appConfig: AppConfig,
+                                        metrics: MetricsService) {
+
+
+  private[connectors] def obligationsUrl(vrn: String): String = {
+    val baseUrl: String = if (appConfig.features.enableVatObligationsService()) {
```

Comment:
This logic is not needed - this is old code taken from the other microservices when we used to either call `vat-api` or `vat-obligations`. We only ever call `vat-obligations` now